Dockerfile

```diff
@@ -1,22 +1,29 @@
-####################################################################################################################################
-# Build stage 1 - Create a distribution of the project which we can copy in the second build stage.
-####################################################################################################################################
-FROM rod_catch_returns/builder AS builder
+ARG PARENT_VERSION=2.3.0-node20.15.0
+
+# https://github.com/DEFRA/defra-docker-node
+FROM defradigital/node:${PARENT_VERSION} AS base
+
+USER root
+
 WORKDIR /app
 
 # Install app dependencies
 COPY / /app
-RUN npm install --ignore-scripts && npm cache clean --force > /dev/null 2>&1
-
-####################################################################################################################################
-# Build stage 2 - Using the distribution from stage 1, build the final docker image with a minimal number of layers.
-####################################################################################################################################
-FROM rod_catch_returns/base
-WORKDIR /app
-COPY --from=builder /app/ /app/
+RUN npm install pm2 -g --ignore-scripts > /dev/null 2>&1 \
+    && npm install --ignore-scripts && npm cache clean --force > /dev/null 2>&1
```

Comment:
why are you suppressing stdout and stderr for installation of pm2 and cache clean, but not for npm install? Don't we want all output, particularly whilst we're developing?

Reply:
I've removed `> /dev/null 2>&1` and the cache clean. I copied those over from rod-licensing. I guess the cache clean was there so when you do a docker build, the next time it runs does a fresh install of the dependencies. With it removed, the npm install runs a lot quicker!